dreampy_dns.py

```diff
@@ -52,10 +52,10 @@ def get_dns_ip(records, protocol='ip'):
     else:
         rec_type = "A"
     for line in records:
-        values = line.expandtabs().split()
```

Comment:
split("\t") separates the line by tabs. This is indeed identical to expandtabs("\t").split(), however the latter goes one step further. If for any reason DH changes the API response and starts using spaces instead of tabs, the code will still work.
You shouldn't need to do this.

Reply:
I agree.  Verified that your code works correctly and that my change is less robust.  Stick with the original.